[WARLOCK S4 DF] Update DF4 GENERATORS FOR WARLOCKS.

Mlklock opened the pull request:
DF4 Warlocks Profiles, some changes may still apply.

Changed Gera for Destro, Aff, Demo

Changed Talent for Destro.

Mlklock committed:
Update DF4_Generate_Warlock.simc

DF4 Warlocks Profiles, some changes may still apply.

seanpeters86 commented on profiles/generators/DF4/DF4_Generate_Warlock.simc:
```diff
@@ -8,21 +8,21 @@
-# trinket1=nymues_unraveling_spindle,id=208615,ilevel=489
-# trinket2=belorrelos_the_suncaller,id=207172,ilevel=489
-# main_hand=iridal_the_earths_master,id=208321,bonus_id=3524,ilevel=489,enchant_id=6655
+# head=attendants_concocting_cover,id=202579,bonus_id=4800/4786/1498/523,ilevel=528,gem_id=192991,enchant_id=7052
+# neck=terross_captive_core,id=195502,bonus_id=8782,ilevel=528,gem_id=192964/192964/192964
+# shoulders=amice_of_the_sinister_savant,id=217214,ilevel=528
+# back=voice_of_the_silent_star,id=204465,bonus_id=4800/4786/1498,ilevel=535
+# chest=cursed_robes_of_the_sinister_savant,id=217215,ilevel=528,enchant_id=6625
+# wrists=vibrant_wildercloth_wristwraps,id=193510,bonus_id=9500/9405/9379/8960/523/8793,ilevel=525,gem_id=192964,crafted_stats=40/49
+# hands=grips_of_the_sinister_savant,id=217211,ilevel=528
+# waist=magathas_spiritual_sash,id=195515,bonus_id=1808,ilevel=528,gem_id=192964,enchant_id=6904
+# legs=leggings_of_the_sinister_savant,id=217213,ilevel=528,enchant_id=6830
+# feet=vibrant_wildercloth_slippers,id=193519,bonus_id=8932/8960/8793,ilevel=525
+# finger1=seal_of_filial_duty,id=195526,bonus_id=4800/4786/1498/523,ilevel=535,gem_id=192964,enchant_id=6550
+# finger2=seal_of_diurnas_chosen,id=195480,bonus_id=1808,ilevel=528,gem_id=192964,enchant_id=6550
+# trinket1=whispering_incarnate_icon,id=194301,bonus_id=4800/4786/1498,ilevel=528
+# trinket2=nymues_unraveling_spindle,id=208615,bonus_id=4800/4786/1520/9576,ilevel=528
+# main_hand=dreambinder_loom_of_the_great_cycle,id=208616,ilevel=528,enchant_id=6655
```
need to remove the comments

Mlklock replied:
Sorry about that.

Mlklock committed:
Update DF4_Generate_Warlock.simc

Ops, forgot about them.

Mlklock committed:
Update DF4_Generate_Warlock.simc

Aff still had 525 crafted gear, its now 522.

Mlklock committed:
Update DF4_Generate_Warlock.simc

https://www.wowhead.com/news/blizzard-on-crafted-item-level-in-dragonflight-season-4-ptr-bug-and-will-be-338499?webhook

525 crafted again.

Mlklock committed:
Update DF4_Generate_Warlock.simc

Updating Trinkets/Gems to reflect the nerfs towards idol and the need for additional crit from its removal.

Mlklock commented:
https://www.raidbots.com/simbot/report/6XCR5R2CLyNAiwdNM1qUn8

https://www.raidbots.com/simbot/report/uxs8sJAdzaiTdHG1C4Jk2B 

https://www.raidbots.com/simbot/report/eLmJS3EGbazYZiTqc8bhFM

Removed the Nerfed Idol Trinket, swapped the gems towards Crit and some gear parts to account for the crit loss.

Mlklock committed:
Update DF4_Generate_Warlock.simc

Changing Destro Talent.

Mlklock reviewed (commented):
Some Changes due to Trinket nerf, others are just improvements i found around.
